src/components/UI/dropdown/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState, useRef } from 'react';
+import Link from 'next/link';
+
```

Comment:
This space can be removed and similiarly in other files as well

Reply:
okay removed that also.